### ISSUE TITLE
Autosend COT forwarding issue

### ISSUE THREAD

**Issue by whiskas85** [paulmandal/atak-forwarder#194]
I have 3 nodes:
- A: Connected to SERVER
- B: Connected to SERVER + Meshtastic Forwarder
- C: Connected Mesttastic Forwarder

If I set a COT in AUTOSEND on:
- C: everyone receive it (Node B forward it to node A)
- B: everyone receive it
- A: just B receive it. Node B do not forward it to Meshtastic Plugin

The same happens for positioning

**Comment by CypressXt** [comment]
Hey @whiskas85, thanks for the report, I've experienced the same on my side :+1: .

**Comment by paulmandal** [comment]
You can imagine the connection between ATAK and a Server (likely over WiFi) as a big pipe, and the connection to Meshtastic as a little straw. You couldn't feed the big pipe of the ATAK->Server connection into the tiny straw of the ATAK->Meshtastic connection without clogging it up.

**Comment by whiskas85** [comment]
Yes I understand the situation. What about bulk update them in one message? Removing all the useless information but keeping ID and position.
Do you think can be possible?
